src/hypofuzz/interface.py

```diff
@@ -95,16 +95,23 @@ def pytest_collection_finish(self, session: pytest.Session) -> None:
                     flush=True,
                 )
                 continue
-
             # Wrap it up in a FuzzTarget and we're done!
             try:
-                # Skip state-machine classes, since they're not
-                if isinstance(item.obj, RuleBasedStateMachine.TestCase):
-                    target = partial(run_state_machine_as_test, item.obj)
+                if hasattr(item.obj, "_hypothesis_state_machine_class"):
+                    # I guess it's conceivable this is possible...I just don't see how!
+                    assert (
+                        extra_kw == {}
+                    ), "Not possible for RuleBasedStateMachine.TestCase to be parametrized (we think?)"
```

Comment:
Correct, because Hypothesis is responsible for creating the `TestCase` object, and it's not collected by pytest iirc.